community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/BatchingTransactionAppender.java

```diff
@@ -318,16 +318,19 @@ protected void forceAfterAppend( LogForceEvents logForceEvents ) throws IOExcept
         // This is okay, however, because unparkAll() spins when it sees a null next pointer.
         ThreadLink threadLink = new ThreadLink( Thread.currentThread() );
         threadLink.next = threadLinkHead.getAndSet( threadLink );
+        boolean iAttemptedForce = false;
```

Comment:
Can we please remove this strange `i` prefix and call variable `forceAttempted` or `attemptedForce` instead?